docs/overview/decorators.md

```diff
@@ -31,30 +31,30 @@ Decorators available for your controllers.
 
 Decorators available for your controller methods.
 
-| Decorator   | Description                                            |
-| ----------- | ------------------------------------------------------ |
-| @httpGet    | Binds a controller method to a GET HTTP verb.          |
-| @httpPost   | Binds a controller method to a POST HTTP verb.         |
-| @httpPut    | Binds a controller method to a PUT HTTP verb.          |
-| @httpPatch  | Binds a controller method to a PATCH HTTP verb.        |
-| @httpHead   | Binds a controller method to a HEAD HTTP verb.         |
-| @httpDelete | Binds a controller method to a DELETE HTTP verb.       |
-| @httpMethod | Binds a controller method to a specified HTTP verb.    |
+| Decorator   | Description                                                        | Usage                          |
+| ----------- | ------------------------------------------------------------------ | ---------------------------- |
+| @httpGet    | Binds a controller method to a GET HTTP verb.                      | @httpGet("/path")            |
+| @httpPost   | Binds a controller method to a POST HTTP verb.                     | @httpPost("/path")           |
+| @httpPut    | Binds a controller method to a PUT HTTP verb.                      | @httpPut("/path")            |
+| @httpPatch  | Binds a controller method to a PATCH HTTP verb.                    | @httpPatch("/path")          |
+| @httpHead   | Binds a controller method to a HEAD HTTP verb.                     | @httpHead("/path")           |
+| @httpDelete | Binds a controller method to a DELETE HTTP verb.                   | @httpDelete("/path")         |
+| @httpMethod | Binds a controller method to a specified HTTP verb.                | @httpMethod("verb", "/path") |
 
 ### Parameter decorators
 
-Decorators available for your controller method parameters.
+Here is a list of all parameter decorators available in Expresso TS, along with their description and usage:
```

Comment:
Instead of Expresso TS is ExpressoTS all together.

Reply:
I'm going to make the modification in all the lines that are "Expresso TS" to "ExpressoTS".